service/src/main/kotlin/fi/espoo/evaka/application/notes/ApplicationNoteQueries.kt

```diff
@@ -17,8 +18,10 @@ fun Database.Read.getApplicationNotes(applicationId: ApplicationId): List<Applic
 SELECT 
     n.id, n.application_id, n.content, 
     n.created, n.created_by, (SELECT name FROM evaka_user WHERE id = n.created_by) AS created_by_name,
-    n.updated, n.updated_by, (SELECT name FROM evaka_user WHERE id = n.updated_by) AS updated_by_name
+    n.updated, n.updated_by, (SELECT name FROM evaka_user WHERE id = n.updated_by) AS updated_by_name,
+    n.message_content_id, m.thread_id as message_thread_id
 FROM application_note n
+LEFT JOIN message m ON m.content_id = n.message_content_id
```

Comment:
This might (in theory) have the effect that duplicate note rows are returned, if this join matches multiple `message` rows. Not sure if this is a problem in practise, because messages related to application are always sent to a single recipient in a single thread, so there's only one `message`.

Reply:
Should never happen™ 😅

But if it does, I'm sure we'll get a bug report that multiple copies of the same note are visible in the UI.